core/scripts/run-tests.sh

```diff
@@ -191,6 +191,103 @@ if ($args['cache']) {
 
 $test_list = simpletest_script_get_test_list();
 
+//Report to ZenCI that testing started.
+if ($args['zenci']) {
```

Comment:
Is there any way that the task of reporting could be part of `core/zen-ci/run_tests.sh`? I don't like needing to tie in vendor-specific code into our otherwise-generic test running script.